internal/cmd/beta/sqlserverflex/user/describe/describe.go

```diff
@@ -0,0 +1,167 @@
+				"$ stackit beta sqlserverflex user describe xxx --instance-id yyy"),
+			examples.NewExample(
+				`Get details of an SQLServer Flex user with ID "xxx" of instance with ID "yyy" in JSON format`,
+				"$ stackit beta sqlserverflex user describe xxx --instance-id yyy --output-format json"),
+		),
+		Args: args.SingleArg(userIdArg, nil),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := context.Background()
+			model, err := parseInput(p, cmd, args)
+			if err != nil {
+				return err
+			}
+
+			// Configure API client
+			apiClient, err := client.ConfigureClient(p)
+			if err != nil {
+				return err
+			}
+
+			// Call API
+			req := buildRequest(ctx, model, apiClient)
+			resp, err := req.Execute()
+			if err != nil {
+				return fmt.Errorf("get SQLServer Flex user: %w", err)
+			}
+
+			return outputResult(p, model.OutputFormat, *resp.Item)
+		},
+	}
+
+	configureFlags(cmd)
+	return cmd
+}
+
+func configureFlags(cmd *cobra.Command) {
+	cmd.Flags().Var(flags.UUIDFlag(), instanceIdFlag, "ID of the instance")
+
+	err := flags.MarkFlagsRequired(cmd, instanceIdFlag)
+	cobra.CheckErr(err)
+}
+
+func parseInput(p *print.Printer, cmd *cobra.Command, inputArgs []string) (*inputModel, error) {
+	userId := inputArgs[0]
+
+	globalFlags := globalflags.Parse(p, cmd)
+	if globalFlags.ProjectId == "" {
+		return nil, &errors.ProjectIdError{}
+	}
+
+	model := inputModel{
+		GlobalFlagModel: globalFlags,
+		InstanceId:      flags.FlagToStringValue(p, cmd, instanceIdFlag),
+		UserId:          userId,
+	}
+
+	if p.IsVerbosityDebug() {
+		modelStr, err := print.BuildDebugStrFromInputModel(model)
+		if err != nil {
+			p.Debug(print.ErrorLevel, "convert model to string for debugging: %v", err)
+		} else {
+			p.Debug(print.DebugLevel, "parsed input values: %s", modelStr)
+		}
+	}
+
+	return &model, nil
+}
+
+func buildRequest(ctx context.Context, model *inputModel, apiClient *sqlserverflex.APIClient) sqlserverflex.ApiGetUserRequest {
+	req := apiClient.GetUser(ctx, model.ProjectId, model.InstanceId, model.UserId)
+	return req
+}
+
+func outputResult(p *print.Printer, outputFormat string, user sqlserverflex.InstanceResponseUser) error {
+	switch outputFormat {
+	case print.JSONOutputFormat:
+		details, err := json.MarshalIndent(user, "", "  ")
+		if err != nil {
+			return fmt.Errorf("marshal SQLServer Flex user: %w", err)
+		}
+		p.Outputln(string(details))
+
+		return nil
+	case print.YAMLOutputFormat:
+		details, err := yaml.MarshalWithOptions(user, yaml.IndentSequence(true))
+		if err != nil {
+			return fmt.Errorf("marshal SQLServer Flex user: %w", err)
+		}
+		p.Outputln(string(details))
+
+		return nil
+	default:
+		table := tables.NewTable()
+		table.AddRow("ID", *user.Id)
+		table.AddSeparator()
+		table.AddRow("USERNAME", *user.Username)
+		if user.Roles != nil && len(*user.Roles) != 0 {
+			table.AddSeparator()
+			table.AddRow("ROLES", strings.Join(*user.Roles, ", "))
```

Comment:
This may look better with a `\n` separator, especially if there are more roles in the future